docs/setup.rst

```diff
@@ -63,6 +63,19 @@ BigQuery access)
 
 Now proceed to `"Setup" <#setup>`__.
 
+Docker Compose
+------
+
+1. Make sure you have a docker machine up and running.
+2. Make sure your current working directory is the root of this github repository.
+3. Run `docker-compose up postgres`
+4. Run `./setup/docker/create_database.sh` (chmod +x this file if necessary).
+    * This will access the postgres container and set up the database.
+5. Run `docker compose up`
+6. Run `docker-compose ls`, take note of the ip for the docker machine you are using, and open the web browser.
```

Comment:
Shouldn't this be `ps` rather than `ls`? The latest version of `docker-compose` doesn't have an `ls` command...